### DISCUSSION TITLE
Organizational Principles

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#4] [Organizational Planning and Structure]
I think we should start talking about organizational principles so we can make sure we're all on the same page as we move forward and bring on more and more people. These are some of my suggested ideals/principles and starting points.

# People over profits
Dating is an important and personal enough activity that it should not be subject to a profit motive. Systems to connect people, e.g. in dating, would ideally be a public good (i.e. something available to all), or at the very least provided at-cost.

# Privacy

# Transparency

# User Equality
All users of the system are equally valuable, have equal basic rights, etc. Essentially, no discrimination based on race, gender, sexual orientation, etc, etc.

We might we want some concept of "mods" at some point, and there might be other ways this could be interpreted in a different way than intended. So we should be clear we are talking about basic human value and rights, etc.

# No Tolerance for Intolerance
Going along with User Equality, we welcome everyone, but your rights only extend to your beneficial interaction with the system and other people in it. This is not a platform for negative engagement with any other group or individual.

**Comment by WilliamDormer** [comment]
Should this discussion be about which people want to see in it rather than the existing ones, since we will probably codify those in a wiki at some point.

**Reply by Oshyan** [reply]
This is a discussion to determine what we codify. Above is my start at it. We don't have a way of doing Wikis as-yet, so any topic here in my view is basically "building toward consensus" on whatever the subject is. If it were possible I'd want 1st post to be editable by all the core team, but I don't know if it is. Lacking that, feel free to respond with your thoughts/changes in comments and I can incorporate. Later on we can move it to Wiki.
